test/mssql_test.cpp

```diff
@@ -433,6 +433,35 @@ TEST_CASE_METHOD(
         REQUIRE(results.get<nanodbc::string>(2) == NANODBC_TEXT("this is varchar max"));
         REQUIRE(results.get<nanodbc::string>(3) == NANODBC_TEXT("this is text"));
     }
+
+    // Unbind all columns
+    {
+        nanodbc::result results = nanodbc::execute(
+            connection,
+            NANODBC_TEXT("select c1, c2, c3, c4 from test_blob_retrieve_out_of_order;")
+        );
+        results.unbind();
+        REQUIRE(results.next());
+        REQUIRE(results.get<int>(0) == 1);
+        REQUIRE(results.get<nanodbc::string>(1) == NANODBC_TEXT("this is varchar max"));
+        REQUIRE(results.get<int>(2) == 11);
+        REQUIRE(results.get<nanodbc::string>(3) == NANODBC_TEXT("this is text"));
+    }
+
+    // Unbind offending column only
+    {
+        nanodbc::result results = nanodbc::execute(
+            connection,
+            NANODBC_TEXT("select c1, c2, c3, c4 from test_blob_retrieve_out_of_order;")
+        );
+        std::vector<short> idx{2};
+        results.unbind(idx);
```

Comment:
Actually, with [respect to my previous comment](https://github.com/nanodbc/nanodbc/pull/236/files#r452966783), does this _just work_ as:

```cpp
results.unbind({2});
```

It's been a few years since I've touched C++ so I'm a bit rusty.

Reply:
Should be, yes